app/domain/authentication/security.rb

```diff
@@ -60,7 +62,11 @@ def validate_user_has_access(req)
 
       # Ensure user has access to the service
       has_access = user_role.allowed_to?('authenticate', webservice_resource)
-      raise NotAuthorizedInConjur, req.user_id unless has_access
+      unless has_access
```

Comment:
codeclimate says to use a guard clause here. This seems kind've silly but might as well do that instead of littering our code with suppression comments. So I believe that would make this:

```
return if has_access

Rails.logger.debug("[OIDC] User '#{req.user_id}' is not authorized to " \
          "authenticate with webservice '#{resource_id}'")
raise NotAuthorizedInConjur, req.user_id
```

Reply:
i really don't like this peace of code. guard clauses are good IMO only when there is one outcome. with more than that it's just an abuse to the code style. don't you think that the current implementation is more readable?

Reply:
Yep, it's much better the way it currently is. Feel free to put a suppression comment in instead.